blaze/server/client.py

```diff
@@ -65,13 +69,17 @@ class Client(object):
 
     blaze.server.server.Server
     """
-    __slots__ = 'url'
+    __slots__ = 'url', 'mode', 'serial'
 
-    def __init__(self, url, **kwargs):
+    def __init__(self, url, mode='json', **kwargs):
         url = url.strip('/')
         if not url[:4] == 'http':
             url = 'http://' + url
         self.url = url
+        if mode not in ('json', 'pickle'):
+            raise ValueError('mode must be either pickle or json')
```

Comment:
I just pushed a commit that generalizes this into a format where the server accepts an iterable of formats, and the client specifies which format it will use. Maybe work can be done to add a series of formats to fall back to.